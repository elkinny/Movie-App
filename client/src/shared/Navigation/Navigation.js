import React from 'react';

import styles from './navigation.scss';

const Navigation = props => {
  return <div className={styles.navigation}>{props.children}</div>;
};

export default Navigation;
