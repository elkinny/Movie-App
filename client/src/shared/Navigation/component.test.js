import Navigation from './Navigation';
import React from 'react';
import { mount } from './../../test/enzymeSetup.js';

describe('Navigation: ', () => {
  it('Navigation to match snapshot', () => {
    const component = mount(<Navigation>Hi</Navigation>);
    expect(component).toMatchSnapshot();
  });
});
