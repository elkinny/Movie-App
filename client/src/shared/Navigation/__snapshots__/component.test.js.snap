// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation:  Navigation to match snapshot 1`] = `
<Navigation>
  <div
    className="navigation"
  >
    Hi
  </div>
</Navigation>
`;
